membrane_curvature/tests/test_mdakit_membcurv.py

```diff
@@ -239,3 +278,14 @@ def test_curvature(md_ref_beads, mdtraj_po4):
 
     for k, k_test in zip(MEMBRANE_CURVATURE_DATA['gaussian_from_z_avg'][lf], K_test[lf]):
         assert_almost_equal(k, k_test)
+
+
+def test_def_all_beads():
+    lipid_types = ['POPC', 'POPE']
+    lfs = ["upper", "lower"]
+    head_list = [4286, 11531, 18891]
+    topology = md.load(GRO_MEMBRANE_PROTEIN).topology
+    beads_test = def_all_beads(lipid_types, lfs, head_list, topology)
+    for lt in lipid_types:
+        for bead, bead_t in zip(MEMBRANE_CURVATURE_DATA['beads']["upper"][lt], beads_test["upper"][lt]):
+            assert_almost_equal(int(bead), int(bead_t))
```

Comment:
As this is an integer, this should be fully equal instead of almost equal (which is usually used for floats). You should also know that both values are already integers, right? Could you `assert bead == bead_t`?